packages/brightdata/api.test.ts

```
@@ -0,0 +1,344 @@
+import { AuthMissingError, logEventFromContext } from 'corsair/core';
+import {
+	AccountEndpoints,
+	ScraperEndpoints,
+	SerpEndpoints,
+	WebUnlockerEndpoints,
+} from './endpoints';
+import {
+	BrightDataEndpointInputSchemas,
+	BrightDataEndpointOutputSchemas,
+} from './endpoints/types';
+import { brightdata, type BrightDataContext } from './index';
```

Biome (2.5.7)

[error] Separate type imports from other imports.

(lint/style/useImportType)